crates/optimism/rpc/src/eth/pending_block.rs

```diff
@@ -45,6 +45,15 @@ where
         )>,
         Self::Error,
     > {
+        if let Some(rx) = self.inner.flashblocks_rx.as_ref() {
+            if let Some(block) = rx.borrow().as_ref() {
+                return Ok(Some((
+                    block.recovered_block.clone(),
+                    Arc::new(block.execution_output.receipts.iter().flatten().cloned().collect()),
```

Comment:
I'd alos like this as a standalone fn of Opethapi

fn `pending_flashblock` or smth